tests/unit/browser/test_qutehistory.py

```diff
@@ -0,0 +1,117 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
```

Comment:
Please rename this file (in a separate commit without any other changes) to `test_qutescheme.py` - we could also put other `qute:*` tests in it in the future.